bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Widget.java

```diff
@@ -2708,5 +2708,15 @@ int getSystemMetrics(int nIndex) {
 	return OS.GetSystemMetrics(nIndex);
 }
 
+boolean adjustWindowRectEx(RECT lpRect, int dwStyle, boolean bMenu, int dwExStyle) {
+	/*
+	 * DPI-dependent version of the method was introduced with Windows 10 Version 1607
+	 */
+	if (OS.WIN32_BUILD >= OS.WIN32_BUILD_WIN10_1607) {
+		return OS.AdjustWindowRectExForDpi (lpRect, dwStyle, bMenu, dwExStyle, getZoom());
```

Comment:
Why does this use `getZoom()`?
- The method expects the DPI, not the zoom, so it probably needs to be converted?
- Why use the device zoom instead of the native zoom, like in the `getSystemMetrics()` method? Is that correct?